manifests/agent/follow.pp

```diff
@@ -39,11 +40,15 @@
     } else {
       fail 'can not specify both, token and destination'
     }
+    
+    if $multi_line_start_pattern != '' {
+      $entry_identifier = "entry_identifier = ${multi_line_start_pattern}\n"
+    }
 
     file { "${confd_path}.conf":
-      content => "# managed by puppet, module ${::module}\n[${clean_title}]\npath = ${my_path}\n${token_or_destination}\n",
+      content => "# managed by puppet, module ${::module}\n[${clean_title}]\npath = ${my_path}\n${token_or_destination}\n${$entry_identifier}",
```

Comment:
This is a bit confusing, could you put the new line in line 49 instead of line 45 to be consistent?